be/src/vec/aggregate_functions/aggregate_function_product.h

```diff
@@ -77,6 +77,30 @@ struct AggregateFunctionProductData<Decimal128> {
     void reset(Decimal128 value) { product = std::move(value); }
 };
 
+template <>
+struct AggregateFunctionProductData<Decimal128I> {
+    Decimal128I product {};
+
+    template <typename NestedType>
+    void add(Decimal<NestedType> value, UInt32 scale) {
+        product *= value;
+        product /= DataTypeDecimal<Decimal128I>::get_scale_multiplier(scale);
```

Comment:
why directly pass `DataTypeDecimal<Decimal128I>::get_scale_multiplier(scale)` to reduce the function call ？